src/client/components/AccordionItem/index.js

```diff
@@ -0,0 +1,37 @@
+import Alert from "react-bootstrap/Alert";
+import { useTranslation } from "react-i18next";
+import PropTypes from "prop-types";
+import React, { useState } from "react";
+
+function AccordionItem(props) {
+  const { header, expandedBody } = props;
+  const { t } = useTranslation();
+
+  const [showDetail, setShowDetail] = useState(false);
+
+  const EN_DASH = "–";
+  return (
+    <React.Fragment>
+      <Alert
+        variant="secondary"
+        className="d-flex toggleAccordion"
+        onClick={() => setShowDetail(!showDetail)}
+        role="button"
+        aria-label={t("retrocerts-confirmation.show-details")}
+      >
+        <div className="flex-fill">
+          <span className="toggleCharacter">{showDetail ? EN_DASH : "+"}</span>
+          {header}
+        </div>
+      </Alert>
+      {showDetail && <div className="detail">{expandedBody}</div>}
+    </React.Fragment>
+  );
+}
+
+AccordionItem.propTypes = {
+  header: PropTypes.element,
+  expandedBody: PropTypes.element,
```

Comment:
Maybe mark these as isRequired?

Reply:
Oh good point, I should do that for the other components I've created too. Thanks!